client/src/utils.ts

```diff
@@ -1,6 +1,41 @@
+import { differenceInMinutes, fromUnixTime, toDate } from 'date-fns';
+
 /**
  * @description Convert micro to stacks.
  * @param amountInMicroStacks - the amount of microStacks to convert
  */
 export const microToStacks = (amountInMicroStacks: string | number) =>
   Number(amountInMicroStacks) / Math.pow(10, 6);
+
+export const numberWithCommas = (x: string | number): string => {
+  let parts = x.toString().split('.');
+  parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  return parts.join('.');
+};
+
+export const getLastBlockTime = (lastBlockTimestamp: number): number => {
+  const currentTimestamp = Date.now();
+
+  return differenceInMinutes(
+    toDate(currentTimestamp),
+    fromUnixTime(lastBlockTimestamp)
+  );
+};
+
+export const numFormatter = (num: number): string | undefined => {
+  if (num > 1000000 && num < 1000000000) {
+    return (num / 1000000).toFixed(0) + 'M';
+  }
+
+  if (num > 1000000000) {
+    return (num / 1000000000).toFixed(0) + 'B';
+  }
+};
+
+export const getNextCycle = (block: number) => {
+  const timeInDays = (block * 10) / 1440;
```

Comment:
can you add a comment explaining what you are doing here please?

Reply:
When looking at the code first question I have, is what are the values "10" and "1440"